docs/source/usersguide/output/statepoint.rst

```diff
@@ -212,33 +206,31 @@ if (run_mode == MODE_EIGENVALUE)
 
             Value for each filter bin of this type.
 
-    **/tallies/tally i/n_nuclides** (*int*)
-
-        Number of nuclide bins. If none are specified, this is just one.
+    **/tallies/tally i/nuclides** (*char[][]*)
 
-    **/tallies/tally i/nuclides** (*int[]*)
-
-        Values of specified nuclide bins (ZAID identifiers)
+        Values of specified nuclide bins.
 
     **/tallies/tally i/n_score_bins** (*int*)
 
-        Number of scoring bins.
+        Number of scores.
```

Comment:
I think this was clearer before as "scoring bins" as opposed to "scores." Perhaps it should be expanded to say that some scores may have more than one scoring bins (*e.g.*, "scatter-PN") for clarity.